irc/handlers.go

```diff
@@ -42,25 +41,54 @@ func accHandler(server *Server, client *Client, msg ircmsg.IrcMessage, rb *Respo
 		return accRegisterHandler(server, client, msg, rb)
 	} else if subcommand == "verify" {
 		rb.Notice(client.t("VERIFY is not yet implemented"))
+	} else if subcommand == "unregister" {
+		return accUnregisterHandler(server, client, msg, rb)
 	} else {
 		rb.Add(nil, server.name, ERR_UNKNOWNERROR, client.nick, "ACC", msg.Params[0], client.t("Unknown subcommand"))
 	}
 
 	return false
 }
 
+// ACC UNREGISTER <accountname>
```

Comment:
Hmm, I'm a bit iffy on having this command as an actual command-command.

Could we leave this as a ChanServ command instead and forgo making it an explicit `ACC` subcommand for now? Mostly, for changes to `ACC` I'd like to ensure we develop the changes along with the spec, since we're approaching it that way for now. (In addition to this, above in commands.go we'll want to re-up the minimum params from 2 to 3 for `ACC`).

Reply:
Makes sense, will do.